brain_node:
  ros__parameters:
    game:
      team_id: 29 # consistant with GameControl
      player_id: 0 # 0 | 1 | 2 | 3
      field_type: "adult_size" # adult_size | kid_size 
      player_role: "striker" # striker | goal_keeper
      player_start_pos: "left" # right | left
      
    robot:
      robot_height: 1.0
      odom_factor: 1.2
      vx_factor: 0.90
      yaw_offset: 0.10
      joystick: "logitech" # logitech | beitong

    rerunLog:
      enable: false
      server_addr: "127.0.0.1:9876"
      img_interval: 1
